.github/vpn/dev.ovpn

```diff
@@ -1,6 +1,6 @@
 client
-remote azuregateway-baa20077-683b-4e57-bc65-6d65264ded69-f09662ccc71f.vpn.azure.com 443
-verify-x509-name 'baa20077-683b-4e57-bc65-6d65264ded69.vpn.azure.com' name
+remote azuregateway-22cf0077-cb28-403f-9ba2-6803b055350f-2c4e2059782c.vpn.azure.com 443
+verify-x509-name '22cf0077-cb28-403f-9ba2-6803b055350f.vpn.azure.com' name
```

Comment:
I had to completely tear down my dev environment when testing the Palo Alto, so my VPN address changed when I redeployed.